pontoon/machinery/openai_service.py

```diff
@@ -28,6 +28,7 @@ def get_translation(
             - Clarity is Key: Ensure the translation conveys the original message in the clearest possible manner, without ambiguity or unnecessary complexity.
             - Consistent Simplicity: Maintain a consistent level of simplicity throughout the translation.
             - Preserve All HTML Tags: Ensure that all HTML tags present in the original text are retained in the translation.
+            - Avoid adding closing punctuation if it's not present in the original text.
```

Comment:
Made this change because I've noticed over time that it tends to add punctuation at the end of strings that don't have it in the source.